riptide-compression/src/test/java/org/zalando/riptide/compression/GzipClientDriver.java

```diff
@@ -0,0 +1,58 @@
+package org.zalando.riptide.compression;
+
+import com.github.restdriver.clientdriver.ClientDriver;
+import com.github.restdriver.clientdriver.DefaultRequestMatcher;
+import com.github.restdriver.clientdriver.exception.ClientDriverSetupException;
+import com.github.restdriver.clientdriver.jetty.ClientDriverJettyHandler;
+import com.github.restdriver.clientdriver.jetty.DefaultClientDriverJettyHandler;
+import com.google.gag.annotation.remark.Hack;
+import org.eclipse.jetty.server.Server;
+import org.eclipse.jetty.server.ServerConnector;
+import org.eclipse.jetty.server.handler.gzip.GzipHandler;
+
+@Hack
+final class GzipClientDriver extends ClientDriver {
+
+    static ClientDriver create() {
+        return new GzipClientDriver(new DefaultClientDriverJettyHandler(new DefaultRequestMatcher()));
+    }
+
+    private int port;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/app/whiskeysierra/riptide/pullRequest?prid=4501367)